### DISCUSSION TITLE
Hot Warm Cold Storage of Logs

### DISCUSSION THREAD

**Discussion by cerberus10748** [Security-Onion-Solutions/securityonion#3007] [Unsupported Versions]
Hello,  
I am spinning up my distributed environment for my enterprise and am trying to devise a method to store alot of logs. I need to have 90 Days of Hot logs to access for compliance as well as store 1 year cold or in long term storage. Anyone ever set up this type of scenario? If so how did you set up storage to migrate old logs off of Hot into Cold.

**Comment by weslambert** [comment]
You can tag the indices based on age, etc with Curator, and they will move off to the correct node, when appropriate.

**Reply by cerberus10748** [reply]
So In that case, I would spin up a storage node in my distributed environment and then tell curator to move them to that storage node correct?

**Reply by weslambert** [reply]
In short, yes; however, this is not an OOB configuration, so we would not be able to help configure/support it.
